app/services/CheckoutService.scala

```diff
@@ -75,7 +75,12 @@ class CheckoutService(
     } yield {
       updateAuthenticatedUserDetails()
       sendETDataExtensionRow(result)
+      subscribe.paymentMethodOpt match {
+        case Some(CreditCardReferenceTransaction(customer)) =>
```

Comment:
I think it's also worth asking whether we should be pushing the Direct Debit details into Salesforce as well